examples/BET/DFDCtoFlow360BET.py

```diff
@@ -32,9 +32,9 @@ def getBetDiskParams(diskIdx):
     # All these parameters are explained in the Flow360 documentation under
     # https://docs.flexcompute.com/projects/flow360/en/latest/solverConfiguration/solverConfiguration.html?highlight=initialBladeDirection#betdisks-list
 
-    # mesh is in inches so meshUnit needs to be 0.0254m per in ( i.e. per mesh unit). DFDC inputs are in metric system.
+    # mesh is in inches so meshUnit needs to be 39.37in per m ( i.e. per mesh unit). DFDC inputs are in metric system.
```

Comment:
it is a bit confusing, when dealing with mm user provides 0.001, see comment:

> meshUnit is in m, if your grid is in mm then meshUnit = 0.001 etc...

So why here is inverse of it? I would expect user provides 0.0254 when working with inches.
Can we call the parameter `meshUnitInMeters` then it will be clear what user needs to input?